src/routes/areas/updateAreas.test.ts

```diff
@@ -0,0 +1,49 @@
+import { FastifyInstance, RouteOptions } from 'fastify';
+import createServer from '../../server';
+import updateArea from './updateArea';
+import AreaModel from '../../models/area';
+
+
+describe('PUT /area', () => {
+  let server: FastifyInstance;
+  beforeAll(() => {
+    server = createServer({ authRoute: [updateArea] as RouteOptions[] });
+  });
+
+  it('returns a 204 status code when updates an area correctly', async () => {
+    const areaId = 2;
```

Comment:
Why areaId is 2? 🤔 
Are we assuming that another test created another area previously? If so, I prefer to use the ID of the response. What happens if I only run this test? 🤔